From 001b3c1cab4321bf85f8900525f5b20f9653b1c2 Mon Sep 17 00:00:00 2001
From: Richard Zhu <hongxing.zhu@nxp.com>
Date: Tue, 27 Nov 2018 16:25:16 +0800
Subject: [PATCH 5218/5242] MLK-20472 arm64: dts: correct the pad
 configurations of pcie

commit  4de7601b8264f98600e069fc2e866ef6987c31a9 from
https://source.codeaurora.org/external/imx/linux-imx.git

Correct the pad confirations of the pcie perst and epdev_on
on 8qm/qxp platforms.

Signed-off-by: Richard Zhu <hongxing.zhu@nxp.com>
Reviewed-by: Fugang Duan <fugang.duan@nxp.com>
(cherry picked from commit 28d5b68c1fa7568a2444915b71fab12e8a2d4350)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8q-arm2.dtsi  |    4 ++--
 .../dts/freescale/fsl-imx8qm-mek-domu-dpu1.dts     |    4 ++--
 .../boot/dts/freescale/fsl-imx8qm-mek-domu.dts     |    4 ++--
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dtsi |    2 +-
 arch/arm64/boot/dts/freescale/fsl-imx8x-arm2.dtsi  |    2 +-
 5 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8q-arm2.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8q-arm2.dtsi
index d0bedaa..e28a0e7 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8q-arm2.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8q-arm2.dtsi
@@ -527,7 +527,7 @@
 			fsl,pins = <
 				SC_P_PCIE_CTRL0_CLKREQ_B_LSIO_GPIO4_IO27	0x04000021
 				SC_P_PCIE_CTRL0_WAKE_B_LSIO_GPIO4_IO28		0x04000021
-				SC_P_PCIE_CTRL0_PERST_B_LSIO_GPIO4_IO29		0x04000021
+				SC_P_PCIE_CTRL0_PERST_B_LSIO_GPIO4_IO29		0x06000021
 			>;
 		};
 
@@ -535,7 +535,7 @@
 			fsl,pins = <
 				SC_P_PCIE_CTRL1_CLKREQ_B_LSIO_GPIO4_IO30	0x04000021
 				SC_P_PCIE_CTRL1_WAKE_B_LSIO_GPIO4_IO31		0x04000021
-				SC_P_PCIE_CTRL1_PERST_B_LSIO_GPIO5_IO00		0x04000021
+				SC_P_PCIE_CTRL1_PERST_B_LSIO_GPIO5_IO00		0x06000021
 			>;
 		};
 
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu-dpu1.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu-dpu1.dts
index 0a17c36..b5b8b3b 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu-dpu1.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu-dpu1.dts
@@ -777,9 +777,9 @@
 			fsl,pins = <
 				SC_P_PCIE_CTRL0_CLKREQ_B_LSIO_GPIO4_IO27	0x04000021
 				SC_P_PCIE_CTRL0_WAKE_B_LSIO_GPIO4_IO28		0x04000021
-				SC_P_PCIE_CTRL0_PERST_B_LSIO_GPIO4_IO29		0x04000021
+				SC_P_PCIE_CTRL0_PERST_B_LSIO_GPIO4_IO29		0x06000021
 				SC_P_LVDS1_I2C0_SDA_LSIO_GPIO1_IO13		0x06000000
-				SC_P_USDHC2_RESET_B_LSIO_GPIO4_IO09		0x04000021
+				SC_P_USDHC2_RESET_B_LSIO_GPIO4_IO09		0x06000021
 			>;
 		};
 
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
index 7e79aa8..4fe6a55 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
@@ -931,9 +931,9 @@
 			fsl,pins = <
 				SC_P_PCIE_CTRL0_CLKREQ_B_LSIO_GPIO4_IO27	0x04000021
 				SC_P_PCIE_CTRL0_WAKE_B_LSIO_GPIO4_IO28		0x04000021
-				SC_P_PCIE_CTRL0_PERST_B_LSIO_GPIO4_IO29		0x04000021
+				SC_P_PCIE_CTRL0_PERST_B_LSIO_GPIO4_IO29		0x06000021
 				SC_P_LVDS1_I2C0_SDA_LSIO_GPIO1_IO13		0x06000000
-				SC_P_USDHC2_RESET_B_LSIO_GPIO4_IO09		0x04000021
+				SC_P_USDHC2_RESET_B_LSIO_GPIO4_IO09		0x06000021
 			>;
 		};
 
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dtsi
index d89109a..f084335 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dtsi
@@ -598,7 +598,7 @@
 
 		pinctrl_pcieb: pcieagrp{
 			fsl,pins = <
-				SC_P_PCIE_CTRL0_PERST_B_LSIO_GPIO4_IO00		0x04000021
+				SC_P_PCIE_CTRL0_PERST_B_LSIO_GPIO4_IO00		0x06000021
 				SC_P_PCIE_CTRL0_CLKREQ_B_LSIO_GPIO4_IO01	0x04000021
 				SC_P_PCIE_CTRL0_WAKE_B_LSIO_GPIO4_IO02		0x04000021
 			>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8x-arm2.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8x-arm2.dtsi
index 52c4179..363f7e8 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8x-arm2.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8x-arm2.dtsi
@@ -456,7 +456,7 @@
 
 		pinctrl_pcieb: pciebgrp{
 			fsl,pins = <
-				SC_P_PCIE_CTRL0_PERST_B_LSIO_GPIO4_IO00		0x04000021
+				SC_P_PCIE_CTRL0_PERST_B_LSIO_GPIO4_IO00		0x06000021
 				SC_P_PCIE_CTRL0_CLKREQ_B_LSIO_GPIO4_IO01	0x04000021
 				SC_P_PCIE_CTRL0_WAKE_B_LSIO_GPIO4_IO02		0x04000021
 			>;
-- 
1.7.9.5

